configure.in

```
@@ -556,6 +556,7 @@ WX_ARG_WITH(gnomevfs,      [  --with-gnomevfs         use GNOME VFS for associat
 WX_ARG_WITH(libnotify,     [  --with-libnotify        use libnotify for notifications], wxUSE_LIBNOTIFY)
 WX_ARG_WITH(opengl,        [  --with-opengl           use OpenGL (or Mesa)], wxUSE_OPENGL)
 WX_ARG_WITH(xtest,         [  --with-xtest            use XTest extension], wxUSE_XTEST)
+WX_ARG_WITH(nanosvg,       [  --with-nanosvg          use NanoSVG for rasterizing SVG], wxUSE_NANOSVG)
```

I think this should be
```suggestion
WX_ARG_WITH(nanosvg,       [  --without-nanosvg          don't use NanoSVG for rasterizing SVG], wxUSE_NANOSVG)
```
because it is used by default.

---

All the external libraries above have `--with-lib` format, and are enabled / searched for by default. Why not keep this the same?

---

My reasoning was that we could fail to find these libraries, so for them this could somewhat make sense, but NanoSVG is always available, so it doesn't seem to be very useful to explicitly enable it.

But maybe you're right and it's better to keep it consistent even if it's a bit different. OTOH we do have `--without-gtkprint` already. Oh well, I guess we can leave it like this if you prefer it, there is no real consistency here anyhow.

On a tangentially related reason, the behaviour that would make most sense to me would be to have `--with-foo` require foo and fail if it's not available, `--without-foo` not use it and default (or some `--with-foo=auto`) behave as `--with-foo` does now. But this would almost certainly run into compatibility problems and break builds that worked just fine before because some exotic dependency (like libmspack or something) is not available. So in practice the only thing we could do would be to extend `WX_ARG_WITH` to allow specifying `--with-foo=required`.